### ISSUE TITLE
Starboard Exporter: project branch name and commit hash missing from release v0.3.2

### ISSUE THREAD

**Issue by elchenberg** [giantswarm/giantswarm-catalog#22]
When I try to use the latest release of the Starboard Exporter I get the following error:

```
STDERR:
  Error: UPGRADE FAILED: failed to create resource: NetworkPolicy.extensions "starboard-exporter-network-policy" is invalid: [metadata.labels: Invalid value: "[[ .SHA ]]": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?'), metadata.labels: Invalid value: "[[ -Branch ]]": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')]
```

With release v0.3.1 the values.yaml contains a branch name and commit hash:

```shell
wget -O starboard-exporter-0.3.1.tgz https://github.com/giantswarm/giantswarm-catalog/raw/master/starboard-exporter-0.3.1.tgz
tar xzf starboard-exporter-0.3.1.tgz starboard-exporter/values.yaml --strip-components=1
grep -A2 -E '^project' values.yaml
```

Output:

```yaml
project:
  branch: "HEAD"
  commit: "42d958c65083d64d1ad39235da81e64f5848362d"
```

When I repeat the commands with the v0.3.2 release the output looks different:

```shell
wget -O starboard-exporter-0.3.2.tgz https://github.com/giantswarm/giantswarm-catalog/raw/master/starboard-exporter-0.3.2.tgz
tar xzf starboard-exporter-0.3.2.tgz starboard-exporter/values.yaml --strip-components=1
grep -A2 -E '^project' values.yaml
```

Output:

```yaml
project:
  branch: "[[ .Branch ]]"
  commit: "[[ .SHA ]]"
```

**Comment by stone-z** [comment]
Conversation moved over to the source repo: https://github.com/giantswarm/starboard-exporter/issues/87

**Comment by stone-z** [comment]
This should now be fixed in [v0.3.3](https://github.com/giantswarm/starboard-exporter/releases)
